nestkernel/connection_manager.cpp

```diff
@@ -1462,3 +1472,28 @@ nest::ConnectionManager::unset_have_connections_changed( const thread tid )
     have_connections_changed_[ tid ].set_false();
   }
 }
+
+
+void
+nest::ConnectionManager::collect_compressed_spike_data( const thread tid )
+{
+  if ( use_compressed_spikes_ )
+  {
+    if ( not sort_connections_by_source_ )
+    {
+      throw KernelException( "Spike compression requires sort_connections_by_source to be true." );
+    }
+
+#pragma omp single
+    {
+      source_table_.resize_compressible_sources();
+    } // of omp single; implicit barrier
+
+    source_table_.collect_compressible_sources( tid );
+#pragma omp barrier
+#pragma omp single
+    {
+      source_table_.fill_compressed_spike_data( compressed_spike_data_ );
+    }
+  } // of omp single; implicit barrier
```

Comment:
Isn't this the closing bracket of `if (use_compressed_spikes_)`?

Reply:
yes!